src/style/card.scss

```diff
@@ -0,0 +1,82 @@
+.main {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
```

Comment:
let's use right margins for cards, those should have fixed distance between them